x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/hooks/use_latest_vulnerabilities.tsx

```diff
@@ -20,10 +20,11 @@ interface FindingsAggs {
   count: estypes.AggregationsMultiBucketAggregateBase<estypes.AggregationsStringRareTermsBucketKeys>;
 }
 
-export const getFindingsQuery = ({ query }: any) => ({
+export const getFindingsQuery = ({ query, sort }: any) => ({
```

Comment:
1. can we type this object instead of using `any`?
2. might be better if sort is done by the consumer of this function (same as the `query`), instead of passing an array that is being modified inside the function.